src/components/DeleteAccount/helpers.js

```diff
@@ -0,0 +1,61 @@
+import { WebVaultClient } from 'cozy-keys-lib'
+
+const fetchPasswordValidation = async ({ client, t, currentPassphrase }) => {
+  const instanceURL = client.getStackClient().uri
+  const vaultClient = new WebVaultClient(instanceURL)
+
+  try {
+    await vaultClient.login(currentPassphrase)
+
+    const currentPasswordHash = await vaultClient.computeHashedPassword(
+      currentPassphrase
+    )
+
+    await client
+      .getStackClient()
+      .fetchJSON('POST', '/settings/passphrase/check', {
```

Comment:
Isn't this redundant with `vaultClient.login()`?

Reply:
si comme on disait, mais le login est nécessaire, et on a mis en place la route de la stack, on va garder le comportement initialement conçu. Je fais remonter l'info par contre.